api/src/wfl/service/cromwell.clj

```diff
@@ -268,6 +268,22 @@
                                    options
                                    labels)))
 
+(defn submit-workflows
+  "Submit one or more workflows to cromwell.
+  Formal Parameters:
+   ENVIRONMENT - Cromwell Deployment Environment
+   WDL         - Workflow WDL to be executed
+   IMPORTS-ZIP - Zip archive of WDL dependencies
+   INPUTS      - Sequence of workflow inputs
+   OPTIONS     - Workflow options for entire batch
+   LABELS      - Labels to apply to each workflow
+
+  Return:
+   List of UUIDS for each workflow as reported by cromwell."
+  [environment wdl imports-zip inputs options labels]
+  ; todo: call the batch endpoint
```

Comment:
Is this TODO happening within this PR or is it a later feature?

Reply:
maybe later. It's not required for this feature. Maybe be an efficiency improvement.